examples/text_to_image/requirements_sdxl.txt

```diff
@@ -0,0 +1,6 @@
+accelerate>=0.16.0
+torchvision
+transformers>=4.25.1
+ftfy
+tensorboard
+Jinja2
```

Comment:
@patrickvonplaten we don't need the `invisible-watermark` library here?

Reply:
In my experiments, it is needed in [this section](https://github.com/okotaku/diffusers/blob/train_lora_sdxl/examples/text_to_image/train_text_to_image_lora_sdxl.py#L1162-L1187).

Reply:
Then let's add it to the requirements.txt too?

Reply:
@okotaku if you want you can keep it, but note that since this PR the invisible-watermark library is not mandatory anymore: https://github.com/huggingface/diffusers/pull/4346